### PR TITLE
Added a new section: Chatrooms

### PR DESCRIPTION
**Pull request by harinij**
Adding to the Communities/Groups section a list of active Slack Teams that provide useful forum for tech chats and discussions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,6 +329,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     Women Hack for Non-Profits (WHFNP) is a volunteer community that enhances the skills of women in technology, supports their development and provides opportunities to explore open source. The group matches skilled women to open source projects that are being developed for non-profits, charities and individuals to create a positive social impact.
 
+  Slack Teams.
```
Comment (harsh183) [inline README.md]: 1. Format it as a proper heading, kind of like `### Slack teams'

2. Maybe we should rename the section to something broader, something like chatrooms, or online discussion rooms? We might add more discord servers, gitter rooms, IRC etc.

3. Add a link to this on the table to contents in the top of the README, and add a link to the top of the page right below the heading (copy paste it from another section)

Comment (harsh183) [reply README.md]: For 3, use `([Go to top](#awesome4girls--))`

File: README.md [hunk 2]
```diff
@@ -329,6 +329,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+  
+  - [Women in Tech](http://witchat.github.io/)
+
+    A safe, confidential space for women who work in technology to chat and support each other. 
+
+  - [Women TechMakers](https://www.womentechmakers.com/)
+
+    The early career slack group is pretty active, apply to join.
```
Comment (harsh183) [inline README.md]: This line is okay, but it could use more information.

File: README.md [hunk 3]
```diff
@@ -329,6 +329,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+  
+  - [Systers](https://anitab.org/systers/)
+
+    A community of the Anita Borg Institute, provides a space to discuss and contribute towards Open Source projects, participate in events and discuss any aspect of Tech.
```
Comment (harsh183) [inline README.md]: Replace 'Tech' with 'technology'

**Comment by harsh183** [comment]
Also let me know if you disagree with any of these comments, I'm sure we can work out something great together.

**Comment by harinij** [comment]
Makes sense to add a new section called "Chatrooms" instead of resuing "Communities", as you said it can be used to broaden it further to other gitter, irc teams etc. Have updated the file now.

File: README.md [hunk 4]
```diff
@@ -329,6 +330,23 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+  
+  Online chatrooms such as slack teams, gitter, IRC etc.
+  
+  - [Women in Tech](http://witchat.github.io/)
+
+    A safe, confidential space for women who work in technology to chat and support each other. 
+
+  - [Women TechMakers](https://www.womentechmakers.com/)
+
+    Community of women in early to mid level career who want to propel their career to senior level. This slack group is pretty active, apply to join.
```
Comment (harsh183) [inline README.md]: More shortened with lesser redundancy, `Early to mid-level career women who want to propel their career to senior level. Apply to join.`

File: README.md [hunk 5]
```diff
@@ -329,6 +330,23 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
...
+  
+  - [Systers](https://anitab.org/systers/)
+
+    A community of the Anita Borg Institute members, provides a space to discuss and contribute towards Open Source projects, participate in events and discuss any aspect of technology.
```
Comment (harsh183) [inline README.md]: A community of Anita Borg Institute members. It provides a space to discuss and contribute towards Open Source projects, participate in events and discuss any aspect of technology.

**Comment by harinij** [comment]
Done

**Comment by harsh183** [comment]
@harinij Great job and thanks for putting all the changes! The new section makes the list feel much more complete now. :)

Looking feel free to add anything else you think could help here.

**Pull request merged**